worlds/cv64/__init__.py

```diff
@@ -0,0 +1,326 @@
+from .locations import CV64Location, get_location_info, verify_locations, get_location_names_to_ids, base_id
+from .entrances import verify_entrances, get_warp_entrances
+from .options import CV64Options, CharacterStages, DraculasCondition, SubWeaponShuffle
+from .stages import get_locations_from_stage, get_normal_stage_exits, vanilla_stage_order, \
+    shuffle_stages, generate_warps, get_region_names
+from .regions import get_region_info
+from .rules import CV64Rules
+from .data import iname, rname, ename
+from ..AutoWorld import WebWorld, World
+from .aesthetics import randomize_lighting, shuffle_sub_weapons, rom_empty_breakables_flags, rom_sub_weapon_flags, \
+    randomize_music, get_start_inventory_data, get_location_data, randomize_shop_prices, get_loading_zone_bytes, \
+    get_countdown_numbers
+from .rom import LocalRom, patch_rom, get_base_rom_path, CV64DeltaPatch
+from .client import Castlevania64Client
+
+
+class CV64Settings(settings.Group):
+    class RomFile(settings.UserFilePath):
+        """File name of the CV64 US 1.0 rom"""
+        copy_to = "Castlevania (USA).z64"
+        description = "CV64 (US 1.0) ROM File"
+        md5s = [CV64DeltaPatch.hash]
+
+    rom_file: RomFile = RomFile(RomFile.copy_to)
+
+
+class CV64Web(WebWorld):
+    theme = "stone"
+
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Archipleago Castlevania 64 randomizer on your computer and connecting it to a "
+        "multiworld.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Liquid Cat"]
+    )]
+
+
+class CV64World(World):
+    """
+    Castlevania for the Nintendo 64 is the first 3D game in the franchise. As either whip-wielding Belmont descendant
+    Reinhardt Schneider or powerful sorceress Carrie Fernandez, brave many terrifying traps and foes as you make your
+    way to Dracula's chamber and stop his rule of terror!
+    """
+    game = "Castlevania 64"
+    item_name_groups = {
+        "Bomb": {iname.magical_nitro, iname.mandragora},
+        "Ingredient": {iname.magical_nitro, iname.mandragora},
+    }
+    location_name_groups = {stage: set(get_locations_from_stage(stage)) for stage in vanilla_stage_order}
+    options_dataclass = CV64Options
+    options: CV64Options
+    settings: typing.ClassVar[CV64Settings]
+    topology_present = True
+    data_version = 1
+
+    item_name_to_id = get_item_names_to_ids()
+    location_name_to_id = get_location_names_to_ids()
+
+    active_stage_exits: typing.Dict[str, typing.Dict]
+    active_stage_list: typing.List[str]
+    active_warp_list: typing.List[str]
+    reinhardt_stages: bool = True
+    carrie_stages: bool = True
+    branching_stages: bool = False
+    starting_stage: str = rname.forest_of_silence
+
+    total_s1s: int = 7
+    s1s_per_warp: int = 1
+    total_s2s: int = 0
+    required_s2s: int = 0
+    drac_condition: int = 0
```

Comment:
```suggestion
    total_s1s: int
    s1s_per_warp: int
    total_s2s: int
    required_s2s: int
    drac_condition: int
```

Reply:
here's some, but I think  above are more  class variables that look they were meant to be either instance variables or constants.